.github/workflows/shared-run-e2e-tests.yaml

```diff
@@ -51,29 +51,26 @@ on:
         type: string
         default: 'main'
       operator_type:
-        description: 'The type of operator [either public or private]'
+        description: 'The type of operator [either public or gcp or azure]'
         type: string
         default: 'public'
-      uid2_e2e_identity_scope:
-        description: 'Environment variable to run the E2E test'
-        required: false
+      gcp_project:
+        description: 'The name of the GCP project'
         type: string
-        default: 'UID2'
-      uid2_e2e_pipeline_operator_type:
-        description: 'Environment variable to run the E2E test'
-        required: false
+        required: true
+      gcp_service_account:
+        description: 'The GCP service account'
         type: string
-        default: 'PUBLIC'
-      uid2_e2e_pipeline_operator_url:
-        description: 'Environment variable to run the E2E test'
-        required: false
+        required: true
```

Comment:
Do these need to be required? They won't be used by public, aws or azure operators